components/mpas-albany-landice/src/mode_forward/Interface_velocity_solver.hpp

```diff
@@ -207,7 +209,7 @@ void createReducedMPI(int nLocalEntities, MPI_Comm& reduced_comm_id);
 
 void importFields(std::vector<std::pair<int, int> >& marineBdyExtensionMap,
                 double const* bedTopography_F, double const* lowerSurface_F, double const* thickness_F,
-    double const* beta_F = 0, double const* stiffnessFactor_F = 0, double const* effecPress_F = 0, double const* muFriction_F = 0, double const* temperature_F = 0, double const* smb_F = 0, double eps = 0);
+    double const* beta_F = 0, double const* stiffnessFactor_F = 0, double const* effecPress_F = 0, double const* muFriction_F = 0, double const* bedRoughness_F = 0, double const* temperature_F = 0, double const* smb_F = 0, double eps = 0);
```

Comment:
@mperego , is this change needed to match the variable name in the calling signature in the cpp file?

```suggestion
    double const* beta_F = 0, double const* stiffnessFactor_F = 0, double const* effecPress_F = 0, double const* muFriction_F = 0, double const* bedRoughnessRC_F = 0, double const* temperature_F = 0, double const* smb_F = 0, double eps = 0);
```

Reply:
It's not needed because what matters to the compiler is the position and type of the parameter, but I think it's easier for debugging purposes to use the same names in function declaration and definition.